packages/driver/src/cross-origin/origin_fn.ts

```diff
@@ -126,7 +132,20 @@ export const handleOriginFn = (Cypress: Cypress.Cypress, cy: $Cy) => {
     })
 
     try {
-      const value = window.eval(`(${fn})`)(args)
+      let value
+
+      if (_.isString(fn)) {
```

Comment:
Nice, we only bundle up origin blocks with cypress.require in them. That should save on network calls.